python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -237,8 +215,13 @@ def __init__(
         self_loop=True,
         dropout=0.0,
         layer_norm=False,
-        max_in_degree=None
+        max_in_degree=None,
     ):
```

Comment:
@jermainewang Did you suggest moving the try-except logic here?

Reply:
The current behavior is fine since it won't break `import dgl` if cugraphlib is not installed.